### PR TITLE
Update GET /api/accounts/:address endpoint - Closes #5250

### LINKED ISSUE #5250: Update GET /api/accounts/:address endpoint
### Description

Add the API endpoint:

- `GET /api/accounts/:address`

according to the following specification:

- It should return decoded account
- Validate if address is base64 address if not `400` should be returned
- If address is not found `404` should be returned

### Motivation

The HTTP API should return accounts by address.

### Acceptance Criteria

- Add functional tests
- Register plugin to core, and check if it returns decoded data with address
- Check if invalid address returns 400 with error message

### Additional Information

Related issues: #5213

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

This PR resolves #5250

### How was it solved?

- By adding controller for accounts
- By adding decoding to JSON in base plugin

### How was it tested?

- run `node test/test_app/index.js`
- send request: `http://localhost:4000/api/accounts/nQFJsJYtRL%2FAip9k1a%2FOtigdf7U%3D` should return account
- send request: `http://localhost:4000/api/accounts/nQFJsJYtRL%2FAip8k1a%2FOtigdf7U%3D` and should return 404 with message
- send request: `http://localhost:4000/api/accounts/-invalid` should fail with 400 and message
- Functional tests also added

### CODE REVIEW

File: framework-plugins/lisk-framework-http-api-plugin/test/functional/account.spec.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ */
+import { Application } from 'lisk-framework';
+import axios from 'axios';
+import {
+	createApplication,
+	closeApplication,
+	getURL,
+} from './utils/application';
+
+describe('Hello endpoint', () => {
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-http-api-plugin/test/functional/account.spec.ts:22]: Accounts

Comment (pablitovicente) [reply framework-plugins/lisk-framework-http-api-plugin/test/functional/account.spec.ts]: Fixed

File: framework-plugins/lisk-framework-http-api-plugin/src/controllers/index.ts [hunk 2]
```diff
@@ -12,4 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import * as accounts from './account';
```
Comment (shuse2) [inline framework-plugins/lisk-framework-http-api-plugin/src/controllers/index.ts:15]: i think `export * as accounts from './account'` should work

File: framework-plugins/lisk-framework-http-api-plugin/src/controllers/account.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		});
+		res.status(200).send(codec.decodeAccount(account));
+	} catch (err) {
+		if (
+			/^Specified key accounts:address:(.*)does not exist/.test(
+				(err as Error).message,
+			)
+		) {
+			res.status(404).send({
+				message: `Account with address '${accountAddres}' was not found`,
```
Comment (shuse2) [inline framework-plugins/lisk-framework-http-api-plugin/src/controllers/account.ts:44]: 404 case also should follow the same error format

File: framework-plugins/lisk-framework-http-api-plugin/src/controllers/index.ts (hunk 2 above)
Comment (pablitovicente) [reply framework-plugins/lisk-framework-http-api-plugin/src/controllers/index.ts]: Done.

**Pull request merged**